besspin/cyberPhys/cyberphyslib/cyberphyslib/demonstrator/director.py

```diff
@@ -86,35 +86,32 @@ class IgnitionDirector:
     ]
 
     hacks2patterns = {
-        canlib.HACK_NONE: ledm.LedPatterns.NOMINAL,# TODO?
-        canlib.HACK_OTA: ledm.LedPatterns.NOMINAL,
+        canlib.HACK_NONE: ledm.LedPatterns.NOMINAL,
+        #canlib.HACK_OTA: ledm.LedPatterns.NOMINAL, # TODO: add pattern for OTA hack?
```

Comment:
Are there not going to be patterns for the OTA or infotainment hacks, or have we not added them yet?

Reply:
We/I decided that for the time being it is sufficient to *not* use LEDs to visualize any non-critical hacks, and only use the hacker kiosk to show that OTA/Info hacks are happening. Mostly I didn't know how to add new patterns. @EthanJamesLew is it difficult to develop/test new patterns?

Reply:
New patterns can be developed by adding entries to [led_strings_comprehensive_tuple_colors.csv](https://github.com/GaloisInc/BESSPIN-Tool-Suite/blob/master/besspin/cyberPhys/cyberphyslib/cyberphyslib/demonstrator/utils/led_strings_comprehensive_tuple_colors.csv)

Reply:
Sounds good.